tests/functional/object/mcg/test_pv_pool.py

```diff
@@ -429,6 +430,7 @@ def test_pv_pool_with_nfs(
         )
 
     @tier2
+    @skip_inconsistent
```

Comment:
@pytest.mark.skip(reason="Skip due to issue #12717")

Reply:
@keesturam  Done, thank you.